### ISSUE TITLE
uvc.OpenError

### ISSUE THREAD

**Issue by valvador92** [pupil-labs/pyuvc#59]
Hi !
I have installed pyuvc after installing libuvc and libjpeg-turbo.
When I try to open the cameras I got this error:

![uvc_issue](https://user-images.githubusercontent.com/33961158/50008607-fedea380-ffb4-11e8-9d79-0ebe925b82b5.png)

It seems to detect the 3 cameras but it can't open it.
It is strange because it is working on an other computer. On this computer I can open the 3 cameras and it recognizes them with their name.

Do you know how I can fix it?
Thank you

**Comment by valvador92** [comment]
it is working with Ubuntu 16.04, python 2.7.12 on my laptop

I have to install on the computer of the lab. The OS is Ubuntu 14.04, python 2.7.6

**Comment by valvador92** [comment]
> it is working with Ubuntu 16.04, python 2.7.12 on my laptop
> 
![image](https://user-images.githubusercontent.com/33961158/50008754-6eed2980-ffb5-11e8-8882-21869dda69e0.png)

> I have to install on the computer of the lab. The OS is Ubuntu 14.04, python 2.7.6

**Comment by valvador92** [comment]
During the installation of libuvc on the lab computer, I got this message in red:

Linking C shared library libuvc.so

Maybe it can help you

**Comment by papr** [comment]
> Linking C shared library libuvc.so

This is an expected output. My guess is an issue with 14.04. Can you download the Pupil bundle and check if it is able to open the cameras?

**Comment by valvador92** [comment]
I have downloaded it. Pupil Capture is working on my Laptop with Ubuntu 16.04.
But with Ubuntu 14.04 on the computer lab, I can't open the software.
So maybe the OS is the issue which is annoying

**Comment by papr** [comment]
Indeed. Maybe you can ask your lab supervisor to get you a computer with an up-to-date operating system. :-) Good luck with your further research. Please close the issue if you have the feeling that all your questions have been answered.

**Comment by valvador92** [comment]
I have a computer up-to-date this time.
Ubuntu 16.04 like my laptop

Pupil Capture is running but can't open the cameras of the headset. The images are gray.

I have installed uvc library regarding the ReadMe. I get this if I using it in a python script:

![screenshot from 2019-01-22 18-09-59](https://user-images.githubusercontent.com/33961158/51552552-1725c600-1e71-11e9-801e-7a0923577adf.png)

**Comment by valvador92** [comment]
That is what I have on my laptop.

![screenshot from 2019-01-22 18-21-24](https://user-images.githubusercontent.com/33961158/51553233-9b2c7d80-1e72-11e9-8647-405e80403a21.png)

**Comment by valvador92** [comment]
I have ckecked, I have 2 different kernel versions:
- Laptop: 4.15.0-43-generic
- Lab computer: 4.4.0-141-generic

Could it be the reason of the problem?

**Comment by papr** [comment]
Your user on the first computer does not have enough permissions. You need to add him to the plugdev group and restart.
